lib/modules/platform/github/index.ts

```diff
@@ -121,10 +121,19 @@ export async function initPlatform({
   gitAuthor,
 }: PlatformParams): Promise<PlatformResult> {
   if (!token) {
-    throw new Error('Init: You must configure a GitHub personal access token');
+    throw new Error(
+      'Init: You must configure a GitHub personal access token or app installation token'
+    );
   }
 
-  platformConfig.isGHApp = token.startsWith('x-access-token:');
+  let prefixedToken: string = token;
+  if (token.startsWith('x-access-token:')) {
+    platformConfig.isGHApp = true;
+    prefixedToken = token;
+  } else if (token.startsWith('ghs_')) {
+    platformConfig.isGHApp = true;
+    prefixedToken = 'x-access-token:' + token;
```

Comment:
don't prefix, identify required place to modify, so we can more easily remove the obsolete prefix later

Reply:
> don't prefix

Do you mean that I should find a way to omit the `x-access-token:` prefix in the `token` entirely?